src/common/Button/Button.tsx

```diff
@@ -1,12 +1,22 @@
-import * as types from 'src/types';
-
 import { FC } from 'react';
 
 import styles from './Button.module.css';
 
-export const Button: FC<types.ButtonProps> = ({ text, onClick }) => {
+interface ButtonProps {
+	text: string;
+	onClick?(
```

Comment:
```suggestion
	onClick?: React.MouseEventHandler<HTMLButtonElement>
```
or I almost every time use
```suggestion
	onClick?: (e: React.MouseEvent<HTMLButtonElement>) => void;
```